test/rewrite_test.go

```diff
@@ -52,6 +53,8 @@ func testMX(t *testing.T, server string) {
 func testEdns0(t *testing.T, server string) {
 	m := new(dns.Msg)
 	m.SetQuestion("example.com.", dns.TypeA)
+	// allow EDNS0
+	m.SetEdns0(4096, true)
```

Comment:
I set EDNS0 with a buffer size only.
What do we expect from the response in OPT ?

At this moment, there is no change of behavior (in comparison to precedent version).

However, I am wondering if we should expect the OPT generated by the REWRITE at query time ...